app/views/scheduled_publish_mailer/publish_email.text.erb

```diff
@@ -0,0 +1,35 @@
+<%= @edition.document_type.label %> ‘<%= @edition.title %>’
+
+<% if @edition.number > 1 -%>
+<%= t("scheduled_publish_mailer.publish_email.details.update",
+  time: @status.created_at.strftime("%l:%M%P").strip,
```

Comment:
If we use a formatting of `"%-l:%M%P"` we don't need to do the strip

Reply:
Sure, good spot!